packages/rum-core/test/error-logging/error-logging.spec.js

```diff
@@ -285,4 +286,61 @@ describe('ErrorLogging', function() {
       expect(apmServer.errorQueue.items.length).toBe(4)
     }
   })
+
+  it('should capture unhandled rejection events', done => {
+    configService.set('flushInterval', 1)
+    errorLogging.registerListeners()
+
+    spyOn(apmServer, 'sendErrors').and.callFake(errors => {
+      expect(errors[0].exception.message).toMatch(reason.message)
+      done()
+    })
+    /**
+     * simulate window.PromiseRejectionEvent event since its not supported by
+     * all browsers
+     */
+    const reason = new Error(testErrorMessage)
+    const event = new CustomEvent('unhandledrejection')
+    event.reason = reason
+    window.dispatchEvent(event)
+  })
+
+  it('should handle multiple reasons for promise rejections', () => {
```

Comment:
Maybe something like `should handle different type of reason for promise rejection`